gammapy/analysis/config/template-1d.yaml

```diff
@@ -1,46 +1,22 @@
 ---
```

Comment:
@Bultako - Can you please remove this `---` at the top of each YAML file.

I think it's optional, if omitted the result is the same, it works.

People will forget to put this all the time, I don't see the point of putting it or teaching users about it, so would prefer to just not use it in our YAML files and examples.

Reply:
Done.

Reply:
Done.